torch/quantization/fx/quantization_patterns.py

```diff
@@ -1308,9 +1319,30 @@ def convert(self,
                 quantized_graph: Graph,
                 node_name_to_scope: Dict[str, Tuple[str, type]],
                 load_arg: Callable,
+                env: Dict[str, Dict[Optional[torch.dtype], Node]],
```

Comment:
I haven't checked myself, but is there a way to avoid this?  This adds additional coupling between the state of prepare/convert and the quantize handler abstraction.

Reply:
I think we may be able to avoid this if we remove "env" and change everything to mutations (just like in prepare), this is possible if we produce reference patterns by default I think, it makes sense to change to that implementation later when we make reference pattern the default path. With the current flow producing fbgemm/qnnpack ops I think it might still be easier to have `env`

Reply:
in convert.py we have

```
            if quantized:
                env[node.name][activation_dtype(qconfig)] = result
            else:
                env[node.name][torch.float] = result
```

it's not ideal to also modify `env` here, as now code readers have to know about two different files to understand what's going on with `env`.  Can we just move the logic which modifies `env` to `convert.py` and gate it by quantize handler type?

Reply:
@vkuzo I looked it again, I think with some workaround we can remove this env from the apis.